gdp/small_batch/gdp_small_batch.py

```diff
@@ -367,20 +367,12 @@ def external_ref(m, x, logic_expr=None):
         for j in m.j:
             if k == ext_var[j]:
                 m.Y[k, j].fix(True)
-                # m.Y_exists[k, j].indicator_var.fix(
-                #     True
-                # )  # Is this necessary?: m.Y_exists[k, j].indicator_var.fix(True).
-                # m.Y_not_exists[k, j].indicator_var.fix(
-                #     False
-                # )  # Is this necessary?: m.Y_not_exists[k, j].indicator_var.fix(True),
+                m.Y_exists[k, j].indicator_var.fix(True)
```

Comment:
This change affects the code behavior. Did you verify that the problems are equivalent? They should have extra constraints.